### PR TITLE
Collect and display the page title for a department's content

### PR DESCRIPTION
**Pull request by leenagupte**
Trello card: https://trello.com/c/q721ivrT

## Description

Get the titles of every piece of content that belongs to an organisation.

## Preview

screen shot 2016-12-02 at 11 39 18

### CODE REVIEW

File: app/models/importers/organisation.rb [hunk 1]
```diff
@@ -2,6 +2,8 @@ class Importers::Organisation
   attr_reader :slug, :batch, :start
   attr_writer :start
 
+  FIELDS = %w(content_id title)
```
Comment (pmanrubia) [inline app/models/importers/organisation.rb]: The name `FIELDS` is not very descriptive. Can you think of a better name?

File: app/models/importers/organisation.rb [hunk 2]
```diff
@@ -13,7 +15,7 @@ def run
     loop do
       result = search_content_items_for_organisation
       result.each do |content_item_attributes|
-        attributes = content_item_attributes.slice('content_id', 'title')
+        attributes = content_item_attributes.slice(*FIELDS)
```
Comment (pmanrubia) [inline app/models/importers/organisation.rb]: I like how you used the `splat` operator here 👍

File: spec/models/importers/organisation_spec.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
```
Comment (leenagupte) [inline spec/models/importers/organisation_spec.rb]: Will do.

**Comment by leenagupte** [comment]
@pmanrubia I have moved the typo fix to the first commit.

I've moved updating the model (schema?) to the commit where the importer is updated, the first place that the `title` column is needed.

**Comment by pmanrubia** [comment]
LGTM
Please, remember to move the constant below the private section (along with `private_const`) as does not describe the interface of the class.

**Comment by leenagupte** [comment]
@pmanrubia I've moved the private_constant as requested, and rebased. Merging...

**Pull request merged**